### PR TITLE
Replace QPG6100 by QPG6105

### PR DESCRIPTION
**Pull request by lucicop**
Ref: 186778

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,16 +10,16 @@ which can be found under:
...
+CHIP examples are available for the QPG6105 Development Kit board.
 All files required are stored in this repository under
 
 ```
-./qpg6100
+./qpg6105
 ```
 
-![QPG6100 DK board](qpg6100/doc/qpg6100.png)
+![QPG6105 DK board](qpg6105/doc/qpg6105.png)
```
Comment (thcu-gp) [inline README.md:22]: Do we have updated images? If not remove

File: examples/evk/README.md [hunk 2]
```diff
@@ -52,18 +52,18 @@ Next to the CHIP Evaluation Kit, the user has to obtain the following items:
...
 * A personal computer (Windows/Linux/Mac) with PuTTY (or an equivalent serial terminal tool) installed and a free USB port.
 
-### Computer and QPG6100 DK board setup
+### Computer and QPG6105 DK board setup
 
-Use the supplied micro USB cable to connect the QPG6100DK board's micro USB connector (marked on the photo with a white circle) to a USB port of the computer.
+Use the supplied micro USB cable to connect the QPG6105DK board's micro USB connector (marked on the photo with a white circle) to a USB port of the computer.
 
-![Power QPG6100 DK](./imgs/qpg6100_power.png "Power QPG6100 DK")
+![Power QPG6105 DK](./imgs/qpg6105_power.png "Power QPG6105 DK")
```
Comment (thcu-gp) [inline examples/evk/README.md:59]: image of a qpg6105?

File: qpg6105/comps/libmbedtls/trng.c [hunk 3]
```diff
@@ -20,8 +20,8 @@
  * INCIDENTAL OR CONSEQUENTIAL DAMAGES,
  * FOR ANY REASON WHATSOEVER.
  *
- * $Change: 183497 $
- * $DateTime: 2021/10/15 15:12:10 $
+ * $Change: 186338 $
```
Comment (thcu-gp) [inline qpg6105/comps/libmbedtls/trng.c:23]: Do we want this internal reference here?

**Review by thcu-gp — CHANGES_REQUESTED** [review]
The image for the qpg6105 does not exist

File: README.md (hunk 1 above)
Comment (lucicop) [reply README.md:22]: We don't have the images yet, but as far as I can tell, the doc folder is not present for qpg6100 either, so it's a TODO since then.

Comment (lucicop) [reply README.md:22]: Will add a note that the image is of a QPG6100DK and will be updated. Also, will keep the original file name.

File: examples/evk/README.md (hunk 2 above)
Comment (lucicop) [reply examples/evk/README.md:59]: Added a note indicating that the images are for QPG6100DK

**Pull request merged**